gumby/experiments/TriblerDispersyClient.py

```diff
@@ -40,21 +40,33 @@ def stop_dispersy(self):
         raise NotImplementedError("Dispersy is stopped using the tribler session in stop")
 
     def start_session(self):
-        logging.error("Starting Tribler Session")
-        self.session_config = self.setup_session_config()
-        self.session = Session(scfg=self.session_config)
+        from twisted.internet import threads
+
+        def _do_start():
+            logging.error("Starting Tribler Session")
+
+            self.session = Session(scfg=self.session_config)
+
+            self.session.start()
```

Comment:
This method returns a deferred, please wait for it using either `yield` or by adding a callback.